include/IVideoDriver.h

```diff
@@ -115,6 +122,7 @@ namespace video
         virtual bool isAllowedTextureFormat(asset::E_FORMAT _fmt) const = 0;
         virtual bool isHardwareBlendableFormat(asset::E_FORMAT _fmt) const = 0;
 
+        virtual IGPUSpecializedShader* createSpecializedShader(const IGPUShader* _unspecialized, const asset::ISpecializationInfo* _specInfo) = 0;
```

Comment:
list of extensions?

also where's your `IGPUShader` creation function?

Reply:
theres no list of extensions yet

Reply:
ok just make a stub struct/class